### DISCUSSION TITLE
Version 3 CDN

### DISCUSSION THREAD

**Discussion by KevinBatdorf** [alpinejs/alpine#1454] [4. General]
The [docs are showing the V2 CDN](https://alpinejs.dev/essentials/installation) at the moment, so if anyone is looking to use the CDN version until that's updated, you can grab it here:

https://cdn.jsdelivr.net/npm/alpinejs@3.x.x/dist/cdn.min.js
https://cdn.jsdelivr.net/npm/alpinejs@3.x.x/dist/cdn.js

or

https://unpkg.com/alpinejs@3.x.x/dist/cdn.min.js
https://unpkg.com/alpinejs@3.x.x/dist/cdn.js

**Comment by jimaek** [comment]
jsDelivr is also an option :) 
https://cdn.jsdelivr.net/npm/alpinejs@3/dist/cdn.min.js
https://cdn.jsdelivr.net/npm/alpinejs@3/

It also includes download stats, including per version distribution.

**Reply by KevinBatdorf** [reply]
I typically use jsDelivr myself actually.

Maybe update your comment to use 3.x.x though?

**Reply by jimaek** [reply]
Updated :)
